charter.md

```diff
@@ -62,14 +64,18 @@ The initial time frame is set to 1 to 3 months after a major Coq release. This t
 
 There shall be some coupling between the Coq major release cycle and the Coq platform major release cycle. However, the frequency of intermediate (minor) releases is still to be determined. What kind of package updates to include in minor platform releases is still to be determined. The versionning scheme is also still undecided and might depend on the other choices regarding the release cycle.
 
-### Quality constrainsts for included packages
+### Quality constraints for included packages
 
 For now, we haven't set any quality constraints beyond the fact that the packages are sufficiently used and stable. But other constraints, such as the existence of complete documentation, might be set in the future.
 
 ### Licensing constraints for included packages
 
 For now, we haven't yet decided whether packages under non-open source licenses (such as CompCert's academic license) will be accepted in the platform. We could even consider imposing some additional reusability criteria for Coq libraries.
 
+### OCaml compatibility
+
+There should be a clear policy on which OCaml versions must be supported by Coq plugins and tools. The easiest solution for platform maintainers would be to support all OCaml versions that Coq itself supports. However, this should be discussed with package maintainers, who can have a need for more recent OCaml versions.
```

Comment:
@MSoegtropIMC this is the relevant section on OCaml versioning. I don't mind if we fix 4.07.1 for this platform, but this should be very clear in both the documentation and the meta OPAM package.

Reply:
Yes, and also contact the respective package owners and explain that this restriction in OCaml versions should be fixed for the next version of the platform.

The platform setup script always sets up a new opam switch which 4.07.1.